EAS/v2/_init.yaml

```diff
@@ -0,0 +1,3 @@
+# A configuration that may be suitable for testing.
```

Comment:
I'm uneasy about mentioning testing. `_init.yaml` is supposed to provide reasonable values for real usage, not for testing (for the values that are common across all sites, with the rest going in _summit.yaml, etc.).

 Configuration for testing belongs with the package being tested.

If you remove that comment here, please also remove it for the other files you added.

Reply:
I added a configuration for testing in the package.

Reply:
Thank you. I suggest removing comment `# A configuration that may be suitable for testing` from all of these new files.

Reply:
I removed all comments, also from the new ESS config file which wasn't put there by me.